Blish HUD/GameServices/ArcDps/Models/Ev.cs

```diff
@@ -1,36 +1,117 @@
 ﻿namespace Blish_HUD.ArcDps.Models {
 
     /// <summary>
-    ///     Infos and data about the combat event. Have a look at the arcdps plugin documentation for how to use it
+    ///     Infos and data about the combat event. For more information see the arcdps plugin documentation.
```

Comment:
We could maybe add a link to the docs here: https://deltaconnected.com/arcdps/api/